src/components/blockHelpers.tsx

```diff
@@ -66,7 +67,7 @@ const sectionSizeStyles = {
     },
   },
   lg: {
-    padding: '7rem 0',
+    padding: '7.8125rem 0',
```

Comment:
is this intentional?

Reply:
Yes, as there's quite a bit of difference between the two. 7rem = 112px, 7.8125rem = 125px. I always try to think about the "px" size first, and the consider what rems we want, e.g is 125px a good number or does it feel arbitrary/random? If not, then we should apply the rem that gives us 125px, no matter the number. Though a compromise can be made when taking the multiplication table into consideration, though such compromise should always stem from the perspective of a nice px, and not rem.

Reply:
i think we should abandon rems if we "mentally" convert them into pixels anyways tho? and use media queries + px instead